Libplanet/Net/Swarm.cs

```diff
@@ -1443,6 +1452,36 @@ CancellationToken cancellationToken
                 spent);
         }
 
+        private async Task BroadcastBlockAsync(
+            TimeSpan broadcastBlockInterval,
+            CancellationToken cancellationToken)
+        {
+            const string fname = nameof(BroadcastBlockAsync);
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                try
+                {
+                    await Task.Delay(broadcastBlockInterval, cancellationToken);
+                    if (BlockChain.Tip is { } tip)
+                    {
+                        BroadcastBlock(tip);
+                    }
+                }
+                catch (OperationCanceledException e)
+                {
+                    _logger.Warning(e, $"{fname}() is canceled.");
```

Comment:
```suggestion
                    _logger.Warning(e, "{FName}() is canceled.", fname);
```
Serilog suggests that the log message is compile time constant.

Reply:
IMO, `fname` doesn't need to be treated with structured logging because it is a constant value.